pinot-integration-test-base/src/test/java/org/apache/pinot/integration/tests/BaseClusterIntegrationTest.java

```diff
@@ -586,11 +586,13 @@ protected void stopKafka() {
    * Get current result for "SELECT COUNT(*)".
    *
    * @return Current count start result
-   * @throws Exception
    */
-  protected long getCurrentCountStarResult()
-      throws Exception {
-    return getPinotConnection().execute("SELECT COUNT(*) FROM " + getTableName()).getResultSet(0).getLong(0);
+  protected long getCurrentCountStarResult() {
+    return getCountStarResult(getTableName());
+  }
+
+  protected long getCountStarResult(String tableName) {
+    return getPinotConnection().execute("SELECT COUNT(*) FROM " + tableName).getResultSet(0).getLong(0);
```

Comment:
why this change?

Reply:
Currently yes, but this can be a commonly used method, so added it to the base class for sharing.